include/oneapi/dpl/pstl/omp/util.h

```diff
@@ -153,6 +154,90 @@ __process_chunk(const __chunk_metrics& __metrics, _Iterator __base, _Index __chu
     __f(__first, __last);
 }
 
+// abstract class to allow inclusion in __thread_enumerable_storage as member without requiring explicit template
+// instantiation of param types
+template <typename _StorageType>
+class __construct_by_args_base
+{
+  public:
+    virtual ~__construct_by_args_base() {}
```

Comment:
```suggestion
    virtual ~__construct_by_args_base() = default;
```

Reply:
This change usually strongly recommended for `virtual` destructors in the base class.